fastlane/Gemfile

```diff
@@ -22,3 +22,5 @@ end
 
 plugins_path = File.join(File.dirname(__FILE__), 'fastlane', 'Pluginfile')
 eval(File.read(plugins_path), binding) if File.exist?(plugins_path)
+
+gem 'gh-issues-inspector', git: "https://github.com/orta/gh-issues-inspector", branch: "fixed-issue-search"
```

Comment:
Do we know if this branch is going to be merged into master of `gh-issues-inspector`? What happens if that branch gets deleted for people that are trying to run `bundle install` and it's not there?